aiohttp/cookiejar.py

```diff
@@ -125,13 +134,8 @@ def clear(self, predicate: Optional[ClearCookiePredicate] = None) -> None:
                 del self._expirations[(domain, path, name)]
             self._cookies[(domain, path)].pop(name, None)
 
-        next_expiration = min(self._expirations.values(), default=self._max_time)
-        try:
-            self._next_expiration = next_expiration.replace(
-                microsecond=0
-            ) + datetime.timedelta(seconds=1)
-        except OverflowError:
-            self._next_expiration = self._max_time
+        next_expiration = min(self._expirations.values(), default=self.MAX_TIME)
+        self._next_expiration = min(next_expiration + 1, self.MAX_TIME)
```

Comment:
This seems to be about twice as fast:
```suggestion
        self._next_expiration = min(*self._expirations.values(), self.MAX_TIME-1) + 1
```

Reply:
It leads to an error if `self._expirations` is empty: `TypeError: 'int' object is not iterable`.
I read the code twice and found it never actually used anywhere, may we just remove it?

Reply:
The usage of `self._next_expiration` has been removed since e65f1a9eba0ebde848bf0e50aa9be4753334b3c8. If it is intended to be a shortcut of `_do_expiration`, it should be added back.

Reply:
@Rongronggg9 please see the discussion in https://github.com/aio-libs/aiohttp/issues/7790

Reply:
@bdraco Thanks for your information! I didn't notice that.
So for the time being I decided to just keep it :)

Reply:
You can just unpack into a tuple, and it'll work with 0-length lists:
`self._next_expiration = min((*self._expirations.values(), self.MAX_TIME-1)) + 1`

Reply:
The if statement seems to be a tiny bit faster though, so may as well stick with that.